sqlglot/dialects/bigquery.py

```diff
@@ -328,6 +328,7 @@ class Parser(parser.Parser):
 
         FUNCTIONS = {
             **parser.Parser.FUNCTIONS,
+            "COUNTIF": exp.CountIf.from_arg_list,
```

Comment:
Can you instead do this so that other dialects that support the `COUNTIF` variant can benefit?
```python
class CountIf(AggFunc):
    _sql_names = ["COUNT_IF", "COUNTIF"]
```

Reply:
you mean change the original definition of `CountIf` in expressions.py?

Reply:
That's right, change it to the above and get rid of the `FUNCTIONS` entry.